src/DomainHierarchy.ts

```diff
@@ -197,7 +198,8 @@ export class DomainHierarchy {
     const logs = await provider.getLogs(filter);
     const rawLogs = logs.map(log => {
       const parsedLog = contractInterface.parseLog(log);
-      return parsedLog.values;
+      /** ethers_v5 Interface.parseLog incorrectly parses log, so have to use lowlevel alternative */
```

Comment:
Interesting, good comment 👍 . Were you able to find any open issues in ethers repo about this? Or how did you figure this out?

Reply:
I haven't found such issue. Do you think I may create one? I just found `decodeEventLog` in the documentation, where `parseLog` mentioned as more convenient alternative

Reply:
Sure, if you have time to create an issue in the ethers repo, I think that would be a good idea. Not a high priority though.

Reply:
Not sure if this is a bug, so I've started discussion https://github.com/ethers-io/ethers.js/discussions/1842